### PR TITLE
Solved button disappearing issue #289

### PR DESCRIPTION
**Pull request by Rakshit-gen**
image

image

for both the themes

### CODE REVIEW

File: app/patron/patreon-info.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export function PatreonInfo({ session }: { session: { username: string } }) {
                         </p>
                         <a
                             target="_blank"
-                            rel="noreferrer"
+                            rel="noopener"
```
Comment (zoglo) [inline app/patron/patreon-info.tsx:26]: Change seems unrelated to the issue?

File: app/patron/patreon-info.tsx [hunk 2]
```diff
@@ -348,13 +349,13 @@ export const PatreonBunnySvgWithoutLink = ({
         >
             <path
                 // FIXME: Remove
-                // eslint-disable-next-line sonarjs/no-duplicate-string
-                style={{ fill: "var(--bs-secondary)" }}
+                 
+                style={{ fill: "var(--bs-primary)" }}
```
Comment (zoglo) [inline app/patron/patreon-info.tsx:353]: I'd rather have a custom class like this:
```css
button:hover path {
    fill: var(--bs-body-bg)
}
```
image

We shouldn't change the gold bunny (gold splits) to just green splits :)

File: app/home/homepage.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const Homepage = () => {
                         <Link href="/patron">
                             <Button
                                 variant="secondary"
-                                className="btn-lg me-sm-3 px-3 w-160p h-3r fw-medium"
+                                className="btn-lg me-sm-3 px-3 w-160p h-3r fw-medium hover:text-white"
```
Comment (zoglo) [inline app/home/homepage.tsx:25]: We ain't using tailwind but bootstrap 🤔

**Review by zoglo — CHANGES_REQUESTED** [review]

File: app/patron/patreon-info.tsx (hunk 1 above)
Comment (Rakshit-gen) [reply app/patron/patreon-info.tsx:26]: Undid the change

File: app/patron/patreon-info.tsx (hunk 2 above)
Comment (Rakshit-gen) [reply app/patron/patreon-info.tsx:353]: How do I do this, I am a bit confused on how to add styles in the repository.

File: app/home/homepage.tsx (hunk 3 above)
Comment (Rakshit-gen) [reply app/home/homepage.tsx:25]: Added it by mistake while testing, resolved this.

File: app/patron/patreon-info.tsx (hunk 2 above)
Comment (zoglo) [reply app/patron/patreon-info.tsx:353]: We are using Bootstrap 5.3, this + custom are setup in src/styles and compiles on the fly if you are running the server.

File: package-lock.json
Comment (zoglo) [inline package-lock.json:1]: Deleting dependencies seems unrelated? We should ever use npm ci when installing something to not modify it (cross operating system)

File: app/home/homepage.tsx [hunk 4]
```diff
@@ -27,7 +27,7 @@ export const Homepage = () => {
                                 {t("support")} <PatreonBunnySvgWithoutLink />
                             </Button>
                         </Link>
-                        <Link href="/about">
+                        <Link href="/abougt">
```
Comment (zoglo) [inline app/home/homepage.tsx:30]: Newly added error

**Review by zoglo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**